src/service/fixed-layer.js

```diff
@@ -346,20 +346,13 @@ export class FixedLayer {
             }
           }
 
-          // Transferability requires element to be fixed and top or bottom to
-          // be styled with `0`. Also, do not transfer transparent
-          // elements - that's a lot of work for no benefit.  Additionally,
-          // transparent elements used for "service" needs and thus
-          // best kept in the original tree. The visibility, however, is not
-          // considered because `visibility` CSS is inherited. Also, the
-          // `height` is constrained to at most 300px. This is to avoid
-          // transfering of more substantial sections for now. Likely to be
-          // relaxed in the future.
+          // Transferability requires element to be opaque (not 100%
+          // transparent) - that's a lot of work for no benefit. Additionally,
+          // transparent elements used for "service" needs and thus best kept
+          // in the original tree. The visibility, however, is not considered
+          // because `visibility` CSS is inherited.
           const isTransferrable = isFixed && (
-            fe.forceTransfer || (
-              opacity > 0 &&
-                  offsetHeight < 300 &&
-                  (this.isAllowedCoord_(top) || this.isAllowedCoord_(bottom))));
+            fe.forceTransfer || (opacity > 0 && !!(top || bottom)));
```

Comment:
This will return true if top: 1 and bottom: 1, right? I think you want `(!top || !bottom)`.

Reply:
This matches current behavior. You could spec `top: 0; bottom: 0` which passes the old `isAllowedCoord_`.

Reply:
Oh they're strings not numbers.

Ok, for posterity: we only transfer fixed elements that are not auto-positioned to avoid jumping position after transferring to the fixed layer (due to loss of parent positioning context). We could do this work, but we don't (yet).